vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -4051,6 +4391,42 @@ unit:DeciC
   rdfs:label "DeciCoulomb" ;
   prov:wasDerivedFrom unit:C ;
 .
+unit:DeciGM
+  a qudt:Unit ;
+  qudt:conversionMultiplier 1e-01 ;
+  qudt:conversionOffset 0e+00 ;
+  qudt:description "0,000 1-fold of the SI base unit kilogram" ;
+  qudt:hasQuantityKind quantitykind:Mass ;
+  qudt:iec61360Code "0112/2///62720#UAB076" ;
+  qudt:uneceCommonCode "DG" ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
+  rdfs:label "DeciGM" ;
+  skos:prefLabel "decigram" ;
+.
+unit:DeciL
+  a qudt:Unit ;
+  qudt:conversionMultiplier 1e-01 ;
```

Comment:
0.1 Liters, but 1e-4 m**3, so should be 1e-4.

Reply:
fixed